commands/command.py

```diff
@@ -65,8 +57,17 @@ def prune_sessions(session_list):
 
 
 def highlight_names(source_character, in_string, self_color, others_color):
-    # TODO(RF): Define internally to posecolors?
-    # This function is used in tailored_msg.
+    def add_color_tag(name_list, color, text):
+        def replace(match):
+            return "|" + color + match.group(1) + "|n"
+
+        sorted_name_list = sorted(name_list, key=len, reverse=True)
+
+        match_list = "|".join([re.escape(name) for name in sorted_name_list])
+
+        # find a match in the match_list that's not in the middle of a word
+        return re.sub("(?i)(?<![A-Za-z])(%s)(?![A-Za-z])" % match_list, replace, text)
```

Comment:
wow I don't really know what's going on here but whoever came up with this must be REALLY smrt 🙄